docs/AnaplanSink-sparksink.md

```diff
@@ -0,0 +1,24 @@
+# Anaplan sink
+
+ Description
+ -----------
+ Sink plugin for supporting data import from CDF to Anaplan server.
+
+ The placeholder with the same name as in the `Server file` filed must be predefined in the given Anaplan model.
```

Comment:
is 'filed' a typo? (filed -> file?)

Reply:
It would also be good to document that this plugin does not write data in parallel so may take some time on larger datasets.

Are there any size restrictions for an Anaplan file?

Reply:
Thanks, the doc is fixed. No there is no restrictions, but there is a general good practice that to limit the updated file not more than 1 GB.